### ISSUE TITLE
Enforce unique Category display names

### ISSUE THREAD

**Issue by svonworl** [dockstore/dockstore#4581]
The display name of a Category is used throughout the UI, and should uniquely identify a specific Category.

**Describe the solution you'd like**
Enforce the uniqueness of category display names, via a database constraint and checks in the endpoints that create or update categories.

**Describe alternatives you've considered**
Do nothing, and let the curator(s) manage the display names.

**Comment by denis-yuen** [comment]
> Do nothing, and let the curator(s) manage the display names.

 True, the nice thing about these is that we/curators control them

**Comment by melainalegaspi** [comment]
The curator can do this for now.
